.expeditor/config.yml

```diff
@@ -13,34 +13,24 @@ rubygems:
 release_branch:
   - ffi
 
-github:
-  # This deletes the GitHub PR branch after successfully merged into the release branch
-  delete_branch_on_merge: true
-  # The tag format to use (e.g. v1.0.0)
-  version_tag_format: "win32-mmap-{{version}}"
-  # allow bumping the minor release via label
-  minor_bump_labels:
-    - "Expeditor: Bump Minor Version"
+subscriptions:
+  # These actions are taken, in order they are specified, anytime a Pull Request is merged.
+  - workload: pull_request_merged:{{github_repo}}:{{release_branch}}:*
+    actions:
+      - built_in:bump_version:
+          ignore_labels:
+            - "Expeditor: Skip Version Bump"
+            - "Expeditor: Skip All"
+      - bash:.expeditor/update_version.sh:
+          only_if: built_in:bump_version
+      - built_in:update_changelog:
+          ignore_labels:
+            - "Expeditor: Exclude From Changelog"
+            - "Expeditor: Skip All"
+      - built_in:build_gem:
+          only_if: built_in:bump_version
 
-changelog:
```

Comment:
This `changelog` section needs to remain

Reply:
I have added that